### DISCUSSION TITLE
9.0.0 not working anymore with a create-react-app env

### DISCUSSION THREAD

**Discussion by 0rsa** [wojtekmaj/react-pdf#1832] [Q&A]
Hello,
I upgraded to the latest 9.0.0
I use a create-react-app environment

When I build the application, the .mjs generated file imports the *.js based on the local files system

> import A from"C:/dev/projects/node_modules/@babel/runtime/helpers/esm/get.js";
> import e from"C:/dev/projects/node_modules/@babel/runtime/helpers/esm/getPrototypeOf.js";

The consequence is that the library is not working when you deploy the only build on a server
Do you have any tip?
Thank you

**Comment by maiiiiiiii** [comment]
curious - have you tried upgrading to the latest 9.2.1 and CRA 5.0.1? Did you get the same error?

**Comment by wojtekmaj** [comment]
Hi! Given that this problem is only reproducible on CRA which makes me think it's a CRA configuration issue, and that CRA has been officially pronounced dead, I'm afraid my official suggestion to you would be to just switch to Vite. I realize you might be frustrated reading this, but we're not going to get ANY sort of support from CRA maintainers, and it will NEVER receive an update. You'd be better off switching.
